project_name/celery.py

```diff
@@ -8,9 +8,11 @@
 
 from celery import Celery
 
+from project_name.celerybeat_schedule import CELERYBEAT_SCHEDULE
```

Comment:
Shouldn't it be {{project_name}} in order to replace with the real project name?